protos/feast/core/Store.proto

```diff
@@ -103,7 +105,18 @@ message Store {
     //
     BIGQUERY = 2;
 
-    // Unsupported in Feast 0.3
+    // Cassandra stores entities as a string partition key, feature as clustering column.
+    // NOTE: This store currently uses max_age defined in FeatureSet for ttl
+    //
+    // Columns:
+    // - entities: concatenated string of feature set name and all entities' keys and values
```

Comment:
Should we provide a bit more description how the concatenation is performed or the **format/structure** of the key?
And if there are multitple fields for the entity what are the **separator** when they are joined and do they follow the order they are defined in the featureSetSpec?

With our current approach of using **text/string** for the key, we may have some issue though if the type of one of the entity fields is e.g. float or double because different runtimes (Java/Python) or platform may have different way or precision when representing such value as string. I think we can address this later.

Reply:
Another good point. Main motivation for storing as text is a human-readable format. 
As you said, I think this can be addressed later. I will leave a TODO.

Reply:
It'd be nice if the key format is documented by nice, succinct unit tests for a (ideally static) utility that generates and parses them. We discussed this, but we wanted to refactor Redis implementation to follow the same common pattern (and extract these to some module shareable by ingestion and serving, so some current code duplication is removed). So we thought we should do a follow-up refactoring PR for this, possibly in the context of storage modularization. I'll log an issue for this now.

About portable strings, I can't say it'd never happen, but I should hope that entities with a float or double key value are an extreme edge case…

Reply:
Yes sounds good!